src/AppInstallerCLICore/Command.cpp

```diff
@@ -811,6 +811,10 @@ namespace AppInstaller::CLI
         {
             ExecuteInternal(context);
         }
+        if (context.Args.Contains(Execution::Args::Type::Wait))
+        {
+            system("pause");

```

Comment:
Isn't `system` a (mild) crime against humanity?

I've done this before with `getchar()`, which just takes the next character in the console. How is the prompting done for agreements? Maybe that would work too.

Reply:
I wasn't aware that it was a crime against humanity

Reply:
I wouldn't call it a crime against humanity, but we certainly don't need to invoke another process to wait on.  I would also suggest using a similar pattern to `Reporter::PromptForBoolResponse` as @jedieaston suggested.  I would prefer though a `Reporter::PromptForAnyKey` that changes the semantics.